1_fetch/unzip_file.py

```diff
@@ -1,16 +1,16 @@
 import zipfile
 import os
 
-def unzip_file(file_path, out_dir):
-    with zipfile.ZipFile(file_path, 'r') as zip_ref:
+def unzip_file(zip_file_path, out_dir):
```

Comment:
I think these naming changes make sense! Can you also apply these changes to the base repo that the user would be starting from?
https://github.com/USGS-R/snakemake-introduction/blob/main/1_fetch/unzip_file.py